docs/sources/tempo/setup/operator/object-storage.md

```diff
@@ -43,6 +41,72 @@ Tempo Operator supports [AWS S3](https://aws.amazon.com/), [Azure](https://azure
         type: s3
   ```
 
+### Secure Token Service (STS) Installation
+
+* Create an Object Storage secret with keys as follows:
+
+    ```console
+    kubectl create secret generic tempostack-dev-s3 \
```

Comment:
The secret creation step should be at the end as we do not have the role ARN yet.